src/hooks/useServerUpdateRequiredMonitor.ts

```diff
@@ -56,14 +56,7 @@ const useServerUpdateRequiredMonitor = (draftSpecs: DraftSpecQuery[]) => {
                                     return true;
                                 }
 
-                                // Do a quick simple disabled check before comparing the entire object
-                                if (
-                                    resourceConfigs[bindingUUID].meta
-                                        .disable !==
-                                    getDisableProps(binding?.disable).disable
-                                ) {
-                                    return true;
-                                }
+                                // Do no longer check disable here because disable directly updates the draft
```

Comment:
Small grammatical error in this comment: _No longer check ..._ or _Do not check ..._ are suitable corrections.